content/enterprise_influxdb/v1.8/administration/anti-entropy.md

```diff
@@ -286,15 +286,17 @@ If a data node suddenly disappears due to a catastrophic hardware failure or for
 
 _View the [Replacing Data Nodes](/enterprise_influxdb/v1.8/guides/replacing-nodes/#replacing-data-nodes-in-an-influxdb-enterprise-cluster) documentation for instructions on replacing data nodes in your InfluxDB Enterprise cluster._
 
-### Replacing a machine that is running a data node
+### Replacing a machine running a data node
 
-Perhaps you are replacing a machine that is being decommissioned, upgrading hardware, or something else entirely.
-The Anti-Entropy service will automatically copy shards to the new machines.
+To replace a machine running a data node, start the Anti-Entropy service to copy shards to the new machines by doing one of the following:
+
+- For nodes with more than ≈20 GB of data on disk, ?
```

Comment:
For nodes with more data, what's best practice? Removing and re-adding the data node?